kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -453,6 +453,26 @@ def static(path):
         return ""
 
 
+@library.global_function
```

Comment:
Instead of using a Jinja function for loading we could either use the django-webpack-loader or an even better approach could be to use the `HtmlWebpackPlugin` to inject a HTML snippet inside the base template. For the latter, in order to accommodate bundling we could use a single JS file that will dynamically load whatever each page needs. It requires some tweaking in the templates and our JS code to use exports but I believe this way the front-end it's more loosely coupled and will probably be beneficial for the search UI rewrite if we choose to proceed with a framework.